props/prop_base.h

```diff
@@ -721,9 +726,9 @@ class PropBase : CommandParser, Looper, protected SaberBase {
     } else {
       v = diff.len();
     }
-    // If we're spinning the saber, require a stronger acceleration
-    // to activate the clash.
-    if (v > CLASH_THRESHOLD_G + fusor.gyro().len() / 200.0) {
+    // If we're spinning the saber or if loud sounds are playing, 
+    // require a stronger acceleration to activate the clash.
+    if (v > (CLASH_THRESHOLD_G + fusor.gyro().len() / 200.0) + (dynamic_mixer.audio_volume() * (AUDIO_CLASH_SUPPRESSION_LEVEL * 0.000001))) {    
```

Comment:
This needs some line-breaking to make the line shorter.
(Probably after the plusses)

Reply:
added line break